chapters/functions.tex

```diff
@@ -2398,6 +2398,11 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   {\lstinline!annotation(SourceDirectory="modelica:/ModelicaLibraryName/Resources/Source")!}\annotationindex{SourceDirectory}, gives the location for source files.
   The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the source directory, see \cref{external-resources}.
   It is not specified how they are built.
+\item
+  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the function.
+  The license files are human-readable files containing the license conditions.
+  Top-level classes may also contain this annotation (with similar contents), see \cref{license-texts}.
+  When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.
```

Comment:
Rely on `\cref{license-texts}` and avoid repetition:
```suggestion
```

Reply:
This has been removed and the proposed line above added.